src/ffi/pystate.rs

```diff
@@ -21,43 +25,45 @@ extern "C" {
     #[cfg_attr(docsrs, doc(cfg(not(PyPy))))]
     pub fn PyInterpreterState_Delete(arg1: *mut PyInterpreterState);
 
-    #[cfg(all(Py_3_9, not(PyPy)))]
+    #[cfg(all(any(not(Py_LIMITED_API), Py_3_9), not(PyPy)))]
     #[cfg_attr(docsrs, doc(all(Py_3_9, not(PyPy))))]
-    pub fn PyInterpreterState_Get() -> *mut PyInterpreterState;
+    pub fn PyInterpreterState_Get(arg1: *mut PyInterpreterState) -> *mut PyInterpreterState;
 
-    #[cfg(all(Py_3_8, not(PyPy)))]
+    #[cfg(all(any(not(Py_LIMITED_API), Py_3_8), not(PyPy)))]
     #[cfg_attr(docsrs, doc(all(Py_3_8, not(PyPy))))]
-    pub fn PyInterpreterState_GetDict() -> *mut PyObject;
+    pub fn PyInterpreterState_GetDict(arg1: *mut PyInterpreterState) -> *mut PyObject;
 
-    #[cfg(all(Py_3_7, not(PyPy)))]
+    #[cfg(all(any(not(Py_LIMITED_API), Py_3_7), not(PyPy)))]
     #[cfg_attr(docsrs, doc(all(Py_3_7, not(PyPy))))]
-    pub fn PyInterpreterState_GetID() -> i64;
+    pub fn PyInterpreterState_GetID(arg1: *mut PyInterpreterState) -> i64;
 
-    #[cfg(not(PyPy))]
-    #[cfg_attr(docsrs, doc(cfg(not(PyPy))))]
+    #[cfg(not(Py_LIMITED_API))]
     pub fn PyState_AddModule(arg1: *mut PyObject, arg2: *mut PyModuleDef) -> c_int;
-    #[cfg(not(PyPy))]
-    #[cfg_attr(docsrs, doc(cfg(not(PyPy))))]
-    pub fn PyState_RemoveModule(arg1: *mut PyModuleDef) -> c_int;
+
+    #[cfg(not(Py_LIMITED_API))]
+    pub fn PyState_RemoveModule(arg2: *mut PyModuleDef) -> c_int;
```

Comment:
These APIs are definitely available on the limited API as far aw we care, the header suggests that they're available on limited API since Python 3.3.

Also, if these are available for PyPy we should have the PyPy link names set. However from my own checks I couldn't find them, hence the `not(PyPy)` I'd previously written.

Reply:
I checked that `PyState_RemoveModule` is not in PyPy. Will revert this change.